text/src/autogluon/text/automm/configs/data/default.yaml

```diff
@@ -10,3 +10,4 @@ data:
     convert_to_text: False  # Whether to convert the feature to text.
     scaler_with_mean: True  # Whether to normalize with mean.
     scaler_with_std: True  # Whether to normalize with std.
+  positive_class:  # The name of binary classification's positive class. It's used in computing some metrics, e.g., roc_auc. If not provided, then use label_encoder.classes_[1],
```

Comment:
How about following sklearn and still call it `pos_label`: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html#sklearn.metrics.roc_curve

Reply:
+1 on `pos_label`. What about other names, like the function name `try_to_set_positive_label`, do we want to change it to `try_to_set_pos_label`?

Reply:
I get the rationale for naming it `positive_class` now because we can specify it as a string and need to convert it to `pos_label`.

Reply:
Name changed.